src/pre_defined_classes/simple_programs_taking_numeric_user_input/medium/q10/PiperSaratoga.java

```diff
@@ -0,0 +1,56 @@
+package pre_defined_classes.simple_programs_taking_numeric_user_input.medium.q10;
+
+/*
+A Piper Saratoga is a hobby plane with six seats that has a capacity to hold 1200 pounds on a flight.
+ Make a program that helps this plane's pilot determine how much luggage can be brought on the return flight from
+ a vacation. The program needs to consider the weight of the travellers, the weight of the gas needed for the return
+ flight, and the total weight of the luggage from all travelers for the trip. Finally, the program will display the weight allowable for
+ bringing souvenirs home on the return flight.
+
+Sample output:
+    How much do all the travellers weigh?
+    376
+    How much does the gas weigh for the return flight?
+    700
+    What is the total weight of luggage from all travelers?
+    150
+    You can bring back 324.0 pounds of souvenirs!
+
+    //huh?
+    How much do all the travellers weigh?
+    510
+    How much does the gas weigh for the return flight?
+    800
+    What is the total weight of luggage from all travelers?
+    300
+    You can bring back -10.0 pounds of souvenirs!
+
+    //huh?
```

Comment:
Is the `// huh?` for the negative numbers?

Reply:
My bad. the second `huh?` was a wording problem in the question that I've fixed. Forgot to delete my notes.